### ISSUE TITLE
TorchInductor failing training models tracker

### ISSUE THREAD

**Issue by jansel** [pytorch/pytorch#93777]
Currently, 40 models are passing `torchbench.py -d cuda --inductor --training --float16` mode, this issue tracks the ones that don't work. Note that due to `dropout()` RNG differences we don't expect accuracy checks to work for many models.

We are still using a lot of fallbacks, and some workarounds that slow things down.  See pytorch/torchdynamo#327 

## Failures in inference (that also break training)
- 7 models: pytorch/torchdynamo#519 
- 6 models: pytorch/pytorch#93759

## Remaining failures from training:

Updated 10/12/2022

**TorchBench**:
- [ ] @bdhirsh hf_BigBird - https://github.com/pytorch/pytorch/issues/93425
- [ ] @bdhirsh   mobilenet_v3_large, resnet101e Accuracy issue (not root caused yet) - https://github.com/pytorch/torchdynamo/issues/1896
- [ ] @eellison  - vision_maskrcnn(#623 )
- [x] @eellison  tts_angular - https://github.com/pytorch/torchdynamo/issues/1889
- [ ] @ngimel - AMP - dlrm, timm_efficientdet, vision_maskrcnn - flaky - https://github.com/pytorch/torchdynamo/issues/1897
- [ ] @yanboliang - tacotron - https://github.com/pytorch/torchdynamo/issues/1907
- [ ] @bdhirsh/@ezyang mobilenet_v2_quantized_qat - https://github.com/pytorch/pytorch/issues/93445

**HF and TIMM**:
- [ ] @anijain2305 AllenAiLongformerBase accuracy - https://github.com/pytorch/pytorch/issues/93441
- [ ] hf_Longformer, AllenaiLongformerBase performance - https://github.com/pytorch/pytorch/issues/93525
- [ ] @ezyang hf_Longformer, AllenaiLongformerBase  - https://github.com/pytorch/pytorch/issues/93428
- [x] @desertfire `reciprocal` - pytorch/torchdynamo#1565 
- [x] Maximum - pytorch/torchdynamo#1572 
- [x] @anijain2305 AMP - MBartForConditionalGeneration , PLBartForConditionalGeneration , https://github.com/pytorch/torchdynamo/issues/1654

**Stable Diffusion**:

**Deferred**:
- [ ] @davidberard98 @bdhirsh - moco - https://github.com/pytorch/pytorch/issues/93604
- [x] pytorch/pytorch#93683

## Fixed
- [x] @yanboliang - tacotron2 - pytorch/torchdynamo#1693
- [x] https://github.com/pytorch/torchdynamo/issues/1681
- [x] @ngimel - AMP LayoutLMForSequenceClassification, GPT2ForSequenceClassification - https://github.com/pytorch/torchdynamo/issues/1637
- [x] @ngimel  - DistilBertForMaskedLM, DistilBertForQuestionAnswering, hf_DistilBert (AMP) - https://github.com/pytorch/torchdynamo/issues/1619
- [x] @eellison @bdhirsh Longformers - https://github.com/pytorch/torchdynamo/issues/1887
- [x] @eellison convit_base - https://github.com/pytorch/torchdynamo/issues/1886
- [x] @anijain2305 - AMP cat multiple models - https://github.com/pytorch/torchdynamo/issues/1648
- [x] @yanboliang - yolov3 - pytorch/torchdynamo#1352
- [x] @ngimel   - timm_efficientdet (#1150 )
- [x] @mlazos - pytorch_struct - amp - https://github.com/pytorch/torchdynamo/issues/1368
- [x] `le` - pytorch/torchdynamo#1564 
- [x] @desertfire Allenai - https://github.com/pytorch/torchdynamo/issues/1566
- [x] tts_angular - https://github.com/pytorch/torchdynamo/issues/1567
- [x] @ngimel - pytorch/torchdynamo#1304
- [x] speech_transfomer - https://github.com/pytorch/torchdynamo/issues/1287
- [x] vision_maskrcnn - Error signature same as https://github.com/pytorch/torchdynamo/issues/1287. Somehow minifier is not able to repro this.
- [x] accuracy issues - Background_Matting (seems safe to raise tolerance), fastNLP_Bert - unclear
- [x] @samdow - Already active mode - aot_eager - https://github.com/pytorch/torchdynamo/issues/1287
- [x] @anijain2305 - moco (#1148 )
- [x] @ngimel - swin_base_patch4_window7_224 - https://github.com/pytorch/torchdynamo/issues/1439
- [x] DebertaForMaskedLM, DebertaForQuestionAnswering - same error. Lets revisit after https://github.com/pytorch/torchdynamo/issues/1439
- [x] @yanboliang  - XLNetLMHeadModel - index_put - https://github.com/pytorch/torchdynamo/issues/1356
- [x] YituTechConvBert - pytorch/torchdynamo#1535
- [x] @Chillee, @anijain2305  - workaround in pytorch/torchdynamo#1346 - demucs (#1147 )
- [x] @yanboliang - speech_transformer (#1176 )
- [x] @eellison - AllenaiLongformerBase - https://github.com/pytorch/torchdynamo/issues/1445
- [x] Super_SloMo (#1039 )
- [x] @desertfire - HF accuracy issue affect many models - https://github.com/pytorch/torchdynamo/issues/1450
- [x] @ngimel - TIMM accuracy issue - https://github.com/pytorch/torchdynamo/issues/1455
- [x] @yanboliang  - `rexnet_100` accuracy error and also IMA - https://github.com/pytorch/torchdynamo/issues/1460
- [x] @ngimel - AMP accuracy issue - https://github.com/pytorch/torchdynamo/issues/1461
- [x] @ngimel - mixed_b16_224 - Illegal memory access. Not sure where to start - https://github.com/pytorch/torchdynamo/issues/1358
- [x] @eellison - detectron2_* (12 models, pytorch/torchdynamo#1066 )
- [x] @eellison - tts_angular - accuracy fail. Not sure whats going on. Will retry after https://github.com/pytorch/torchdynamo/issues/1291 is fixed.
- [x] @anijain2305   - coat_lite_min, volo_d1_224 (#1135 ) 
- [x] @anijain2305 - convit_base, DebertaForMaskedLM,  - Dashboard infra - fopenmp error
- [x] DebertaForQuestionAnswering (#1273 )
- [x] dla102, dpn107 - Lets solve this one first -  (https://github.com/pytorch/torchdynamo/issues/1255)
- [x] drq, soft_actor_critic - require grad (#1255)
- [x] fastNLP_Bert (#837 )
- [x] pytorch/torchdynamo#1307
- [x] pytorch/torchdynamo#1279
- [x] vision_maskrcnn (#1200)
- [x] nasnetalarge, pnasnet5large (#965)
- [x] tacotron2 (#1188)
- [x] https://github.com/pytorch/torchdynamo/issues/1145 - triton.language.semantic.IncompatibleTypeErrorimpl: invalid operands of type pointer<fp32> and triton.language.fp32 (PLBartForConditionalGeneration, MBartForConditionalGeneration)
- [x] dlrm (#1023)
- [x] mobilenet_v2_quantized_qat (#1149)
- [x] resnet50_quantized_qat (#1149)
- [x] tacotron2 (#1153 )
- [x] Fails only when max_tiles=2, Triton issue: attention_is_all_you_need_pytorch https://github.com/openai/triton/issues/599
- [x] https://github.com/pytorch/torchdynamo/issues/815
- [x] missing decomp for aten.floor_divide.default (BigBird) (solved by https://github.com/pytorch/torchdynamo/pull/1120)
- [x] pytorch/torchdynamo#559 (confirm fixed?)
- [x] pytorch/torchdynamo#569
- [x] pytorch/torchdynamo#557
- [x] pytorch/torchdynamo#530

cc @ezyang @soumith @msaroufim @wconstab @ngimel @bdhirsh @voznesenskym @yanboliang @penguinwu @anijain2305 @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @peterbell10 @desertfire

**Comment by jansel** [comment]
Out of date
